user-testing/Desktop.md

```diff
@@ -0,0 +1,25 @@
+# Desktop Checklist
+
+### Preparation
+
+- create or get an existing test account ready
+
+
+### Test
```

Comment:
also very important is testing the login-screen, advanced settings etc. -- that rarely gets scrutiny during development whereas cahtting/groups do get some test (dogfooding) use before releases.

Reply:
how do testers get an account that requires advanced settings?